content/en/kubernetes.md

```diff
@@ -7,49 +7,32 @@ tags: ["infrastructure", "", ""]
-like an instance of an app, load balancers, persistent storage, and others together 
-in a way that they can be composed into applications. 
-
-Kubernetes enables automation and extensibility, 
-allowing users to deploy applications declaratively in a reproducible way. 
-Software products and projects in the Kubernetes ecosystem take advantage of that automation and extensibility 
-to extend the Kubernetes [API](/application-programming-interface/). 
-This enables them to leverage Kubernetes’ automation and 
-make their tools more accessible to experienced Kubernetes practitioners.
+Kubernetes, often abbreviated as K8s, is an open source container orchestrator. 
+It automates the lifecycle of containerized applications on modern infrastructures, functioning as a "datacenter operating system" that manages applications across a [distributed system](/distributed-systems/).
+
+Kubernetes schedules [containers](container/) across [nodes](/nodes/) in a [cluster](/cluster/), bundling several infrastructure constructs, sometimes referred to as "primitives." 
+These primitives consist of app instances, load balancers, persistent storage, etc., and are composed into applications.
+
+Kubernetes enables automation and extensibility, allowing users to deploy applications declaratively (see below) in a reproducible way. 
+Other software solutions in the Kubernetes ecosystem take advantage of that automation and extensibility.
+Kubernetes is easily extendable via its [API](/application-programming-interface/), and its automation helps make other tools more accessible to experienced Kubernetes practitioners.
 
 ## Problem it addresses
 
-Infrastructure automation and declarative configuration management have been important concepts for a long time and 
-have only become more pressing as [cloud computing](/cloud-computing/) has gained popularity. 
-As demand for compute resources increases and 
-organizations feel pressured to provide more operational capabilities with fewer engineers, 
-new technologies and working methods are required to meet that demand. 
-Additionally, the rise of cloud computing was paired with [containerization](/containerization/) and 
-organizations that were busy automating more traditional infrastructure needed a mechanism to 
-automate the configuration and deployment of their containers.
+Infrastructure automation and declarative configuration management have been important concepts for a long time, but they have become more pressing as [cloud computing](cloud-computing/) has gained popularity. 
+As demand for compute resources increases and organizations need to provide more operational capabilities with fewer engineers, new technologies and working methods are required to meet that demand. 
+Additionally, the rise of cloud computing along with containerization and organizations busy automating more traditional infrastructure need a mechanism to automate the configuration and deployment of their containers.
```

Comment:
what does "organizations busy automating more traditional infrastructure" mean here?

Reply:
@CathPag 
It is ambiguous to me too. Maybe it is redundant as well considering 

> organizations need to provide more operational capabilities with fewer engineers, new technologies and working methods are required to meet that demand

mentioning a similar issue.

Reply:
Ok, so we just drop the sentence entirely, right?

Reply:
I think that's what you meant, so I deleted it. Let me know if that works.